README.md

```diff
@@ -1 +1,41 @@
-# silverstripe-forager-subsites
\ No newline at end of file
+# silverstripe-forager-subsites
+
+Due to the way that filtering works with (eg) Elastic Enterprise Search, you may want to split
+each subsite's content into a separate engine. To do so, you can use the following
+configuration:
+
+```yaml
+SilverStripe\Forager\Service\IndexConfiguration:
+  indexes:
+    content-subsite0:
+      subsite_id: 0
```

Comment:
Perhaps now would be a good time to standardise this back to camel case (IE: `subsiteId` or `subsiteID`)